modules/ROOT/partials/cdm-prerequisites.adoc

```diff
@@ -2,15 +2,15 @@ Read the prerequisites below before using the Cassandra Data Migrator.
 
 * Install or switch to Java 11.
 The Spark binaries are compiled with this version of Java.
-* Select a single VM to run this job and install https://archive.apache.org/dist/spark/spark-3.5.1/[Spark 3.5.1] there.
-No cluster is necessary.
-* Optionally, install https://maven.apache.org/download.cgi[Maven] 3.9.x if you want to build the JAR for local development.
+* Select a single VM to run this job and install https://archive.apache.org/dist/spark/spark-3.5.3/[Spark 3.5.3] there.
+No cluster is necessary and is recommended for most one-time migrations however cluster mode is also supported for complex migrations.
```

Comment:
This sentence is unclear. 

"No cluster is necessary" - that part is clear. 

"and is recommended for most one-time migrations" - what is recommended? Not having a cluster? 

"however cluster mode is also supported for complex migrations" - how is this information related to the first part of the sentence, "no cluster is necessary"?

Reply:
Clarified the sentence for the spark cluster requirement.